lib/csv.rb

```diff
@@ -1325,11 +1328,7 @@ def build_fields_converter
       builtin_converters: Converters,
     }
     options = @base_fields_converter_options.merge(specific_options)
-    fields_converter = FieldsConverter.new(options)
-    normalize_converters(@initial_converters).each do |name, converter|
-      fields_converter.add_converter(name, &converter)
-    end
-    fields_converter
+    parse_converters(@initial_converters, options)
```

Comment:
`parse_` is strange for me. Because it doesn't parse converters.
How about `build_fields_converter`?